src/unity/schemas/Accounts.yml

```diff
@@ -1,2 +1,15 @@
 items:
-  $ref: './Account.yml'
+  allOf:
+    - $ref: './Account.yml'
+    - properties:
```

Comment:
operator get accoutns api returns a list of accounts with some nested resource

account
  - billing contact
  - users
  - marketplace subscription